website/src/app/kb/deploy/gateways/readme.mdx

```diff
@@ -28,8 +28,28 @@ responsible for securely routing traffic between Clients and Resources.
+
+In most cases, **no inbound firewall ports should be opened** for Gateways to
+function properly. If your network has a stateless firewall in place, however,
+you'll need to ensure UDP return traffic can reach the Gateway(s). If you're not
+sure, you should err on the side of caution and leave all inbound ports closed
+unless you experience connectivity issues.
+
+See the table below for firewall recommendations for popular cloud providers.
+
+| Provider | Resource type                                                                                                       | Type      | Inbound ports                                                                                                                                                                                                                                                                        |
+| -------- | ------------------------------------------------------------------------------------------------------------------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| AWS      | [Security Groups](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-security-groups.html)                        | Stateful  | None                                                                                                                                                                                                                                                                                 |
+| Azure    | [Network Security Groups](https://learn.microsoft.com/en-us/azure/virtual-network/network-security-groups-overview) | Stateful  | None                                                                                                                                                                                                                                                                                 |
+| GCP      | [Firewall Rules](https://cloud.google.com/firewall/docs/firewalls)                                                  | Stateful  | None                                                                                                                                                                                                                                                                                 |
+| AWS      | [Network ACLs](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-network-acls.html)                              | Stateless | By default, AWS ACLs allow all inbound. If you've modified these, be sure that `UDP 1024-65536` is allowed from source `0.0.0.0/0` so your Clients can connect directly. If you wish to allow **only** Relayed connections, use the source IPs in [relay-ips.json](/relay-ips.json). |
```

Comment:
Just wondering if the two AWS entries should be next to each other?

Reply:
Yeah I had that, I'll move it back